code/game/machinery/kitchen/microwave.dm

```diff
@@ -238,7 +238,7 @@
 	if(stat & (NOPOWER|BROKEN))
 		return
 	start()
-	if (reagents.total_volume==0 && !(locate(/obj) in ((contents - component_parts) - circuit))) //dry run
+	if (reagents.total_volume==0 && !(locate(/obj) in ((contents - component_parts) - circuit))) //dry run // >>>USING LOCATE
 		if (!wzhzhzh(10))
 			abort()
 			return

```

Comment:
Just look at this fucking proc and C R Y

Reply:
i don't understand code but i'm going to C R Y anyways